src/data/gravity.graphql

```diff
@@ -412,6 +511,11 @@ type Query {
   """
   artworks(ids: [ID!]!): [Artwork!]
 
+  """
+  Find an identity verification by id
+  """
+  identityVerification(id: ID!): IdentityVerification
+
```

Comment:
cc @dleve123 did we ever intend for identity verifications to be exposed via MP's schema?

Reply:
This is the v1 schema, might not be relevant.

Reply:
This is for the Gravity component of the composite schema – so not sure about the v1 part of your comment above?

Regardless, @mdole and I are ignoring this type from Gravity as it's legacy/usused: https://github.com/artsy/metaphysics/pull/2280/commits/00d367e1c30e60700625000aa2e052933a76005d. We tested out a graphql-all-the-way-down approach, but ended up fronting identity verifications from Gravity via REST instead.

The auctions team will own removing those types from Gravity, but that's decoupled from this PR now.